public/data/KosyAbone.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "Abone Kosiso",
+  "type": "personal",
+  "bio": "I am a fullStack Web Developer(MERN Stack), a coding instructor and an open source enthusiast. I love understanding how things work under the hood. I enjoy solving complex problems. Kindly connect with me to find out more about me...",
+  "avatar": "https://github.com/KosyAbone.png",
+  "links": [
+    {
+      "name": "Follow me on Github to see my work",
```

You might want to capitalize the `h` here. :)
```suggestion
      "name": "Follow me on GitHub to see my work",
```

---

Thank you. I have effected this change. 
All is good now. Kindly review